Documentation/PageTsconfig/Mod/_WebList/_downloadPresetsUser.tsconfig

```diff
@@ -0,0 +1,13 @@
+page.mod.web_list.downloadPresets {
+    pages {
+        10 {
+            label = Quick overview (customized)
+            columns = uid, title, crdate, slug
```

Comment:
Just joking: And in 3 years it will be an array like `columnsOnly` :D

PS: I have tested it with `title` for `tt_content`. The column will just be skipped, because it does not exists. So no error. Perfect.

Reply:
Well I think you have to put this kind of review on gerrit, not here ;)